settings.php

```diff
@@ -1515,7 +1542,6 @@
         // Add tab to settings page.
         $page->add($tab);
 
-
```

Comment:
Please leave this second empty line in the file untouched. Having bigger spaces helps to keep the overview of all the settings pages and tabs in this huge settings.php file